streams/src/main/java/org/apache/kafka/streams/kstream/SessionWindowedKStream.java

```diff
@@ -210,26 +208,28 @@ KTable<Windowed<K>, Long> count(final Named named,
                                     final Materialized<K, Long, SessionStore<Bytes, byte[]>> materialized);
 
     /**
-     * Aggregate the values of records in this stream by the grouped key and defined sessions.
```

Comment:
The diff starting here is just "bad" and can be "ignored". I just re-ordered method, to align method order across different classes. There is no actual change below.

Old older count() - aggregate() - reduce() vs new order count() - reduce() -  aggregate()